site/public/js/forum.js

```diff
@@ -1006,6 +1016,7 @@ function dynamicScrollLoadPage(element, atEnd) {
         load_page_callback = function(content, count) {
             spinner_up.hide();
             arrow_up.after(content);
+            // eslint-disable-next-line eqeqeq
             if (count == 0) {
```

Comment:
While I was unable to trigger this `else` condition, the same behavior would likely occur if `count == 0` is replaced with `count === 0` as the code here is almost identical to the preceding condition.